### ISSUE TITLE
Not sure where else to go regarding API

### ISSUE THREAD

**Issue by Freaketh** [juholehtimaki/choso-plugins#1]
Hey, I am looking for com.theplug.PaistiUtils and I saw in the readme to contact him but theres no method to DM on here and this project isn't listed in their repo's. Found a much older one with PaistiSuite but I am interested in the walking aspects, ie LocalPathFinder. Incredibly useful plugins, little bit too much automation for my taste so I just wanna dumb it down and apply some of this API to other things that interest me. I think I enjoy making plugins more than playing but if there's anything you can tell me me or help out with, I would greatly appreciate it.

Thanks for your time, sorry to mark this as an issue.

**Comment by T3rr0or** [comment]
Sorry for necroing this but same
